spec/features/navigation_bar_spec.rb

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2015 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of version 3 of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE about this file by physical or electronic mail,
+# you may find current contact information at www.suse.com
+require_relative "feature_spec_helper"
+
+RSpec.describe "Navigation Bar Buttons", type: :feature do
+  initialize_system_description_factory_store
+
+  let(:store) { system_description_factory_store }
+
+  before(:each) do
+    Server.set :system_description_store, store
+  end
+
+  context "when showing a system description" do
+    before(:each) do
+      description
+    end
+
+    let(:description) { create_test_description(
+        scopes: ["os", "packages", "repositories", "services"],
+        name: "name",
+        store: store,
+        store_on_disk: true
+      )
+    }
+
+    it "disables buttons whose scope was excluded" do
+      visit("/name")
+
+      within(".scope-navigation") do
+        expect(find_link("UF")[:class]).to include("disabled")
+      end
+    end
+  end
+
+  context "when comparing two system descriptions" do
+    before(:each) do
+      description_a
+      description_b
+    end
+
+    let(:description_a) { create_test_description(
+        scopes: ["os", "packages", "repositories"],
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Indent the first parameter one step more than the start of the previous line.